kahuna/public/js/components/gu-date-range/gu-date-range.js

```diff
@@ -90,7 +90,7 @@ guDateRange.directive('guDateRange', [function () {
             });
 
             $scope.$watch('pikaStartValue', function (pikaStartValue) {
-                var date = pikaStartValue && new Date(pikaStartValue);
+                var date = (pikaStartValue && new Date(pikaStartValue)) || new Date();
```

Comment:
Yip - everything is also working as expected.
Passing an empty string to `Date` should error irrespective, so this code is more typesafe anyway.